ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -10,6 +10,11 @@
 from ocs_ci.ocs.resources.csv import CSV
 from ocs_ci.ocs.resources.packagemanifest import get_selector_for_ocs_operator, PackageManifest
 from ocs_ci.utility import utils
+from tests.helpers import check_local_volume, check_pvs_created
+from ocs_ci.ocs.node import get_typed_nodes
+import os
+import shutil
+import yaml
```

Comment:
import order should be changed here so these lines should be moved above the others